forge-1.14.4/src/main/java/org/dynmap/forge_1_14_4/DynmapPlugin.java

```diff
@@ -253,7 +241,10 @@ public void initializeBlockStates() {
                 	if (statename.length() > 0) {
                 		statename += ",";
                 	}
-                	statename += p.getName() + "=" + bs.get(p).toString();
+                    try {
+                        statename += p.getName() + "=" + bs.get(p).toString();
+                    } catch (IllegalFormatConversionException e){
```

Comment:
This should be logged, no?

Reply:
Could be yes, I only looked at depreciation, not the whole code